### ISSUE TITLE
Mod fails to load when only base game & the 2 dependences are enabled

### ISSUE THREAD

**Issue by mgovea** [HEROgold/bot-batteries-technologies_1.0.0#5]
I failed to launch on the Steam version. I disabled all mods except this one and its two dependencies, and still failed to launch. The important part of the log is below.

Edit: and I just upgraded to mod version 2.0.5 from 2.0.3. 2.0.3 worked fine.

```
   1.320 Loading mod settings bot-batteries-technologies 2.0.5 (settings.lua)
   1.322 Loading mod core 0.0.0 (data.lua)
   1.370 Loading mod base 1.1.110 (data.lua)
   1.605 Loading mod bot-batteries-technologies 2.0.5 (data.lua)
   1.612 Error ModManager.cpp:1625: Failed to load mod "bot-batteries-technologies": ...echnologies__/prototypes/roboport/stacking-roboports.lua:2: module technology.roboport-upgrades not found;  no such file __bot-batteries-technologies__/prototypes/roboport/technology/roboport-upgrades.lua no such file __bot-batteries-technologies__/technology/roboport-upgrades.lua no such file __core__/lualib/technology/roboport-upgrades.lua
stack traceback:
	[C]: in function 'require'
	...echnologies__/prototypes/roboport/stacking-roboports.lua:2: in main chunk
	[C]: in function 'require'
	__bot-batteries-technologies__/data.lua:6: in main chunk
   1.613 Loading mod core 0.0.0 (data.lua)
   1.685 Checksum for core: 3215923532
   1.736 Error ModManager.cpp:1625: Error in assignID: recipe-category with name 'crafting' does not exist.
```

**Comment by HEROgold** [comment]
Accidentally added a line that caused this, this is fixed in 2.0.6

**Comment by mgovea** [comment]
Confirmed mod loads correctly now :)
